src/category_theory/discrete_category.lean

```diff
@@ -52,47 +61,64 @@ end functor
 
 namespace nat_trans
 
-def of_homs {I : Type u₁} {F G : discrete I ⥤ C}
+/--
+For functors out of a discrete category,
+a natural transformation is just a collection of maps,
+as the naturality squares are trivial.
+-/
+def of_function {I : Type u₁} {F G : discrete I ⥤ C}
   (f : Π i : discrete I, F.obj i ⟶ G.obj i) : F ⟶ G :=
 { app := f }
 
-@[simp] lemma of_homs_app  {I : Type u₁} {F G : discrete I ⥤ C} (f : Π i : discrete I, F.obj i ⟶ G.obj i) (i) :
-  (of_homs f).app i = f i := rfl
-
-def of_function {I : Type u₁} {F G : I → C} (f : Π i : I, F i ⟶ G i) :
-  (functor.of_function F) ⟶ (functor.of_function G) :=
-of_homs f
-
-@[simp] lemma of_function_app {I : Type u₁} {F G : I → C} (f : Π i : I, F i ⟶ G i) (i : I) :
-  (of_function f).app i = f i := rfl
+@[simp] lemma of_function_app  {I : Type u₁} {F G : discrete I ⥤ C}
+  (f : Π i : discrete I, F.obj i ⟶ G.obj i) (i) : (of_function f).app i = f i :=
+rfl
 
 end nat_trans
 
 namespace nat_iso
 
-def of_isos {I : Type u₁} {F G : discrete I ⥤ C}
+/--
+For functors out of a discrete category,
+a natural isomorphism is just a collection of isomorphisms,
+as the naturality squares are trivial.
+-/
+def of_function {I : Type u₁} {F G : discrete I ⥤ C}
```

Comment:
I'm not sure `function` is the right name here, it's a `nat_iso` built out of things which aren't functions